src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System.UnitTests/ImageTests.cs

```diff
@@ -28,6 +28,9 @@ public void Setup()
         [DataRow("hibernate", "Images\\sleep.dark.png")]
         [DataRow("empty recycle", "Images\\recyclebin.dark.png")]
         [DataRow("uefi firmware settings", "Images\\firmwareSettings.dark.png")]
+        [DataRow("ip v4 addr", "Images\\networkAdapter.dark.png")]
+        [DataRow("ip v6 addr", "Images\\networkAdapter.dark.png")]
```

Comment:
I'm not sure what these are (sorry, I haven't tried to read C# in a while)

Reply:
These are some tests. Left side is the user inputbthat will be simulated. And right side is the image path we expected for the result.